packages/core/src/prompt-system/PromptAssembler.ts

```
@@ -0,0 +1,425 @@
+          selectedModules = this.options.customSelector(
+            context,
+            availableModules,
+          );
+        } else {
+          selectedModules = this.moduleSelector.selectModules(
+            context,
+            availableModules,
+          );
+        }
+        break;
+      case 'default':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)